test/phonons.jl

```diff
@@ -0,0 +1,124 @@
+using Test
+using DFTK
+using LinearAlgebra
+using ForwardDiff
+using StaticArrays
+
+# ## Helper functions
+# Some functions that will be helpful for this example.
+fold(x)   = hcat(x...)
+unfold(x) = Vec3.(eachcol(x))
+
+const MAX_RADIUS = 1e3
```

Comment:
Don't have these as constants, and include in a testset like in the other tests (so it doesn't pollute the namespace)